### PR TITLE
feat: Move global args to the plan args to increase configurability

### PR DESCRIPTION
**Pull request by laurentluce**
Move global args to the plan args to increase configurability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,43 @@ Simply run
...
+
+<details>
+    <summary>Click to see configuration</summary>
+
+You can configure this package using a JSON structure as an argument to the `kurtosis run` function. The full structure that this package accepts is as follows, with default values shown (note that the `//` lines are not valid JSON and should be removed!):
+
+```javascript
+{
+    // The name to give the new etcd service
+    "etcd_name": "etcd",
```
Comment (laurentluce) [inline README.md:25]: Prefix the arg names with the package name to avoid name clashing when the args dictionary is passed to other packages.

**Review by Peeeekay — APPROVED** [review]

**Comment by laurentluce** [comment]
@adschwartz Is it possible to get write access to this repo so I can merge this PR?

**Comment by adschwartz** [comment]
@laurentluce I just add the engineering team, can you try again?

**Comment by laurentluce** [comment]
> @laurentluce I just add the engineering team, can you try again?

I can see the merge option now, thanks @adschwartz!

**Pull request merged**